test/uitest.c

```diff
@@ -59,9 +59,9 @@ static int test_old()
     char pass[16];
     int ok = 0;
 
-    if ((ui_method =
-         UI_UTIL_wrap_read_pem_callback(test_pem_password_cb, 0)) == NULL
-        || (ui = UI_new_method(ui_method)) == NULL)
+    if (!TEST_ptr(ui_method =
+                    UI_UTIL_wrap_read_pem_callback( test_pem_password_cb, 0))
```

Comment:
I disagree, since it's part of the first argument, not an argument to ``TEST_ptr```
But I did it.